### PR TITLE
improvement: mark bots as such in UI

### PR DESCRIPTION
**Pull request by WofWca**
- **refactor: simplify `navbar-chat-subtitle`**
- **improvement: mark bots as such in UI**

Closes https://github.com/deltachat/deltachat-desktop/issues/4332.

Thanks @r10s for the detailed instructions in https://github.com/deltachat/deltachat-desktop/issues/4332!

Tested with a bot. Works.

### CODE REVIEW

File: packages/frontend/src/components/screens/MainScreen.tsx [hunk 1]
```diff
@@ -360,7 +365,7 @@ function ChatHeading({ chat }: { chat: T.FullChat }) {
           </div>
         </div>
         {subtitle && subtitle.length && (
-          <div className='navbar-chat-subtitle'>{chatSubtitle(chat)}</div>
+          <div className='navbar-chat-subtitle'>{subtitle}</div>
```
Comment (WofWca) [inline packages/frontend/src/components/screens/MainScreen.tsx:368]: FYI this change is unrelated.

**Review by nicodh — APPROVED** [review]
Reviewed by testing & reading

**Comment by r10s** [comment]
thanks a lot, @WofWca
